tests/func/test_get.py

```diff
@@ -179,3 +180,31 @@ def test_get_from_non_dvc_master(tmp_dir, erepo_dir, caplog):
 
     assert caplog.text == ""
     assert (tmp_dir / dst).read_text() == "some_contents"
+
+
+def test_get_url_positive(tmp_dir, erepo_dir, caplog):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("foo", "foo")
+
+    caplog.clear()
+    with caplog.at_level(logging.WARNING, logger="dvc"):
```

Comment:
Hm, why are we testing warnings specifically?

Reply:
I wanted to check that there's no log above `WARNING`, but, I think it's more confusing this way, and I'm probably overdoing here. I'll change it to `ERROR`.